src/sysv/systemd/sssd-kcm.service.in

```diff
@@ -9,7 +9,7 @@ Also=sssd-kcm.socket
 
 [Service]
 Environment=DEBUG_LOGGER=--logger=files
-ExecStartPre=+-/bin/chown -f -R -h root:@SSSD_USER@ @sssdconfdir@
+ExecStartPre=+-/bin/chown -f -R -H root:@SSSD_USER@ @sssdconfdir@
```

Comment:
Hi @mgerstner,

do you think this change is reasonable taking into account that:
1) it requires root to write to /etc (and so to make /etc/sssd a symlink)
2) with -H and -R only command line argument is dereferenced
?

#7781 has some details/reason.

Reply:
The recursion logic (`-R`) of GNU `chown` is already safe against race conditions and symbolic links appearing in the directory tree, that is correct.

If you want to support the `@sssdconfdir` being a symlink and if it reasonable to assume that the location it points to is only root controlled, which is the case for `/etc/sssd`, then passing `-H` should be fine. I would document the reason for passing `-H` here then, though, because it's use is a bit special.

Reply:
Thank you.